code/controllers/subsystem/shuttle.dm

```diff
@@ -178,9 +178,30 @@ SUBSYSTEM_DEF(shuttle)
 			Good luck.")
 			return
 		emergency = backup_shuttle
-	var/srd = CONFIG_GET(number/shuttle_refuel_delay)
-	if(world.time - SSticker.round_start_time < srd)
-		to_chat(user, "The emergency shuttle is refueling. Please wait [DisplayTimeText(srd - (world.time - SSticker.round_start_time))] before trying again.")
+	var/refuel_time = CONFIG_GET(number/shuttle_refuel_min_delay) * 2
+	// condensed names for readability of the actual equation
+	var/aa = 0 // amt_of_crew
+	var/xx = 0 // dead_crew
+	var/vv = 0.5 // mult_at_50_percent
+	var/sensors_detection = 0.15 // this * amt of crew is the amount that needs to actually be dead before we start fucking with the refueling timer or allowing a call to begin with
+
+	for(var/mob/M in GLOB.mob_list)
+		if(ishuman(M))
+			var/mob/living/carbon/human/H = M
+			if(H.mind && H.mind.assigned_role )
+				aa++
+				if(H.stat && !H.suiciding)
+					xx++
+		CHECK_TICK
+	if(!xx || xx < round((aa * sensors_detection)) && round((aa * sensors_detection)) >= 5)
+		to_chat(user, "Central Command Emergency Sensors indicate [station_name()] is operating normally. Your request for a shuttle is denied for the time being. Thank you, and have a productive shift!")
+		return
+
+	var/final_mult = (((min(aa/2,xx))*(1 - ((min(aa/2,xx)) / aa))) / (aa/4)) * vv
```

Comment:
Use better variable names and move some things that can be precalculated out like `min(aa/2,xx)`

Reply:
This is the equation you gave me i wouldn't know what to name that var

Reply:
What do you mean you dont know, you're the one that input the vars into this equation. Just call it whatever maes sense like `normalized_dead` or something